Make.inc

```diff
@@ -827,6 +827,9 @@ $(error "please install either GNU tar or bsdtar")
 endif
 endif
 
+# Do not try to extract owner uids, even if we're root (e.g. in sandboxes)
+TAR += --no-same-owner
```

Comment:
Is this a btar and gtar flag? I would have thought this would be part of our UNTAR instead